caddyconfig/caddyfile/parse.go

```diff
@@ -165,6 +173,12 @@ func (p *parser) begin() error {
 		if err != nil {
 			return err
 		}
+		// Keep track of which tokens come from snippets and what the snippet name is
```

Comment:
Maybe this comment should explain why; right now it doesn't add too much to what is already obvious from the lines below.

Reply:
Done